env/filehandler/filehandler.go

```diff
@@ -0,0 +1,33 @@
+package filehandler
+
+import (
+	"github.com/zouyx/agollo/v3/env"
+	jsonConfig "github.com/zouyx/agollo/v3/env/config/json"
+)
+
+//Suffix 默认文件保存类型
+const Suffix = ".json"
+
+var file FileHandler
+
+var (
+	ConfigFileMap  = make(map[string]string, 1)
+	JsonFileConfig = &jsonConfig.ConfigFile{}
+)
+
+//FileHandler 备份文件读写
+type FileHandler interface {
+	WriteConfigFile(config *env.ApolloConfig, configPath string) error
+	GetConfigFile(configDir string, namespace string) string
+	LoadConfigFile(configDir string, namespace string) (*env.ApolloConfig, error)
+}
+
+//SetFileHandler 设置备份文件处理
+func SetFileHandler(inFile FileHandler) {
```

Comment:
```suggestion
func SetFileHandler(handlerName string,inFile FileHandler) {
```

我觉得可以使用 map 来支持多种格式化类型。